datastore/load_test.go

```diff
@@ -1290,6 +1290,23 @@ func TestLoadNull(t *testing.T) {
 	}
 }
 
+func TestLoadNilInterface(t *testing.T) {
```

Comment:
I think a system test would also be good to have.

Reply:
Added